content/CORE/Storage/Snapshots.md

```diff
@@ -5,39 +5,26 @@ weight: 20
 
 {{< toc >}}
 
-Snapshots are one of the most powerful features of ZFS.
-A snapshot provides a read-only point-in-time copy of a file system or volume.
-This copy does not consume extra space in the ZFS pool.
-The snapshot only records the differences between storage block references whenever the data is modified.
-
-{{< include file="/_includes/SnapshotsSnippet.md" type="page" >}}
-
-{{< expand "Why do I want to keep snapshots?" "v" >}}
-Snapshots keep a history of files and provide a way to recover an older or even deleted files.
-For this reason, many administrators take regular snapshots, store them for some time, and copy them to a different system.
-This strategy allows an administrator to roll the system data back to a specific point in time.
-In the event of catastrophic system or disk failure, off-site snapshots can restore data up to the most recent snapshot.
-{{< /expand >}}
+{{< include file="/_includes/SnapshotsIntroSnippet.md" type="page" >}}
 
 Taking snapshots requires the system have all [pools]({{< relref "/CORE/Storage/Pools/_index.md" >}}), [datasets]({{< relref "" >}}), and [zvols]({{< relref "" >}}) already configured.
 
 ## Creating a Single Snapshot
 
 {{< hint ok >}}
-To save time and regularly create fresh snapshots, consider making a [Periodic Snapshot Task]({{< relref "PeriodicSnapshotTasks.md" >}}) instead.
+Consider making a [Periodic Snapshot Task]({{< relref "/SCALE/DataProtection/_index.md" >}}) to save time and create regular, fresh snapshots.
```

Comment:
This reference needs to point to the CORE content location, not SCALE.

Reply:
issue fixed, PR is currently rebuilding.